xml/System.Diagnostics/ActivitySamplingResult.xml

```diff
@@ -54,7 +54,7 @@
       </ReturnValue>
       <MemberValue>3</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The activity object should be populated the same as the <see cref="F:System.Diagnostics.ActivitySamplingResult.AllData" /> case. Additionally, Activity.IsRecorded is set to <see langword="true" />. For activities using the W3C trace ids, this sets a flag bit in the ID that will be propagated downstream requesting that the trace is recorded everywhere.</summary>
```

Comment:
Folks, the API `Activity.IsRecorded` does not exist. Can you please help me fix this summary with the correct text? Please edit the suggestion in this comment and let me know so I can apply it:

```suggestion
        <summary>The activity object should be populated the same as the <see cref="F:System.Diagnostics.ActivitySamplingResult.AllData" /> case. Additionally, Activity.IsRecorded is set to <see langword="true" />. For activities using the W3C trace ids, this sets a flag bit in the ID that will be propagated downstream requesting that the trace is recorded everywhere.</summary>
```

Reply:
I fixed it and committed too. 

---
In reply to: [488942859](https://github.com/dotnet/dotnet-api-docs/pull/4853#discussion_r488942859) [](ancestors = 488942859)